department-of-veterans-affairs/caseflow
# Queue | Don't wipe issue dispositions during checkout flow

Resolves #5396 

### Description
When an attorney goes to the Draft Decision checkout flow, we un-set issue dispositions, forcing them to set options in the SelectDispositionsView. Sometimes, though, a judge will check out a case to an attorney for review, in which case its issues already have dispositions. If applicable, they should render in the SelectDispositionsView.

### Acceptance Criteria 
- [ ] Go to Draft Decision checkout flow for an appeal with issues with dispositions
- [ ] Confirm disposition dropdowns have those values pre-selected on page load

### Testing Plan
1. Go to `/queue` as `BVAAABSHIRE` (i.e. to view cases that have already been checked out / had dispositions set), start a Draft Decision checkout flow
2. Confirm disposition dropdowns have pre-set values
3. Finalize Draft Decision checkout, updating dispositions, check out to Anjali Abshire (`BVAAABSHIRE`). After returning to `/queue`, reload*, navigate to that appeal, confirm dispositions were updated

![disposition_dropdowns](https://user-images.githubusercontent.com/8533004/39646463-8662d630-4fa9-11e8-9d1e-3c503c2d5d37.gif)

*Reloading is needed here because when finishing a checkout, we take a shortcut and simply delete the appeal from redux, anticipating/mimicking the server response (i.e. checking out to another user). In this case, we're checking an appeal out to the same user, though, so the appeal disappears until the page is reloaded
